packaging/wcm/webapp/src/main/webapp/WEB-INF/conf/wcm-extension/wcm/system-configuration.xml

```diff
@@ -34,6 +34,7 @@
         <description>some params for runtime</description>
 
         <property name="parameterizedPageURI" value="${wcm.config.parameterizedPageURI:/detail}" />
+        <property name="parameterizedURIParam" value="${wcm.config.parameterizedURIParam:content-id}"/>

```

Comment:
This param should be documented if not done @mzorai  @Rdidmohamedelhadi .

Reply:
Ok I will create a new doc issue and will handle it